yaksh/forms.py

```diff
@@ -150,7 +150,8 @@ def save(self):
             new_profile.key_expiry_time = timezone.now() + \
                                         timezone.timedelta(minutes=20)
         new_profile.save()
-        return u_name, pwd, new_user.email, new_profile.activation_key
+        user_activation_key = Profile.objects.get(user=new_user).activation_key
```

Comment:
If `settings.IS_DEVELOPMENT` is set True then `user_activation_key` will be blank entry. which is checked in views to avoid sending mail if `user_activation_key` is blank